include/statisticsmanager.h

```diff
@@ -38,50 +38,32 @@
 #define STATISTICSMANAGER_H__
 
 #include "zmq_lvc.h"
-#include "counter.h"
-#include "accumulator.h"
+#include "snmp_counter_table.h"
+#include "snmp_accumulator_table.h"
 #include "httpstack.h"
 
-#define COUNTER_INCR_METHOD(NAME) \
-  virtual void incr_##NAME() { (NAME).increment(); }
-
-#define ACCUMULATOR_UPDATE_METHOD(NAME) \
-  virtual void update_##NAME(unsigned long sample) { (NAME).accumulate(sample); }
-
 class StatisticsManager : public HttpStack::StatsInterface
 {
 public:
-  StatisticsManager(LastValueCache* lvc);
+  StatisticsManager();
   virtual ~StatisticsManager();
 
-  ACCUMULATOR_UPDATE_METHOD(H_latency_us);
```

Comment:
It feels like this change might be simpler if you'd kept these ACCUMULATOR_UPDATE_METHOD method calls - I think that would mean that the change was limited to StatisticsManager, and it would expose the same `update_H_hss_latency_us` methods to everything else (so the existing UTs, for example, would work unchanged).